### PR TITLE
MM-57175 Tour Point Theme Color Update

### PR DESCRIPTION
**Pull request by matthewbirtch**
#### Summary
This PR changes the tour point color to the newly proposed theme colors. This aims to more clearly differentiate the tour points from the rest of the interface.

Also fixes a couple other minor issues with the onboarding flow (video close button, drafts tag margin).

In addition, this PR changes the buttons in the tour points to use the new `btn` classes and radius variables.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-57175
https://mattermost.atlassian.net/browse/MM-53469
https://mattermost.atlassian.net/browse/MM-57310

#### Screenshots
|  before  |  after  |
|----|----|
| image | image | 
| image  | image | 

#### Release Note
```release-note
Updated theme colors for onboarding tour points.
```

### CODE REVIEW

**Review by yasserfaraazkhan — APPROVED** [review]

**Review by marianunez — APPROVED** [review]
Thanks @matthewbirtch! Looks great 🎉

File: webapp/channels/src/components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.scss [hunk 1]
```diff
@@ -43,15 +44,14 @@
         height: 100vh;
         max-height: 506px;
         padding: 0;
-        border: none;
-        border-radius: 8px;
+        border: none !important;
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.scss:47]: This isn't ideal but i am sure you must have had a valid reason to use it here

**Review by M-ZubairAhmed — APPROVED** [review]

File: webapp/channels/src/components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.scss (hunk 1 above)
Comment (matthewbirtch) [reply webapp/channels/src/components/onboarding_tasks/onboarding_video_modal/onboarding_video_modal.scss]: Agreed. It was kind of a lazy way to override. I've updated this to remove the `!important` rule in the latest commit

**Comment by matthewbirtch** [comment]
@M-ZubairAhmed or @yasserfaraazkhan I don't have authorization to merge, so would appreciate the help to do so when you have a moment

**Pull request merged**